qa/README.md

```diff
@@ -0,0 +1,156 @@
+# Acceptance tests
+
+This module contains acceptance tests for the whole Camunda 8 platform. Covering multiple components, if not all together, integrated.
```

Comment:
❓ Can you explain in a bit more detail what you want to express with this? It's not completely clear to me currently.
> Covering multiple components, if not all together, integrated.

Reply:
I feel like grammarly hit me here. 

What I meant is that we are covering multiple components integrated together. This: "if not all of them", is meant that it can even mean integrating all components together fex as StandaloneCamunda

Reply:
Classic "[...], wenn nicht sogar alle zusammen, [...]" 😅

Reply:
```suggestion
This module contains acceptance tests for the whole Camunda 8 platform, covering multiple or even all components at the same time in an integrated manner.
```